doc/api/cli.md

```diff
@@ -1628,6 +1628,14 @@ This option is only supported on macOS and Windows.
 An `ERR_FEATURE_UNAVAILABLE_ON_PLATFORM` exception will be thrown
 when the option is used on a platform that does not support it.
 
+### --watch-preserve-output
+
+Option to ensure output of stdout is preserved during --watch on restart.
```

Comment:
After thinking a bit more about this, I don't think it's a good idea to say this ensures the stdout is preserved because:

- the user can still clear the console, and this option doesn't disable this.
- it's not just stdout, stderr as well.

As well, let's not call out `--watch`, `--watch-path` is also concerned.

```suggestion
Disable the clearing of the console when watch mode restarts the process.
```